### PR TITLE
Langchain integration

### PR DESCRIPTION
**Pull request by DannyBrock**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 openai
-ragstack-ai==0.3.1
+ragstack-ai==0.7.0
 tiktoken
```
Comment (bhalder) [inline requirements.txt:3]: lanchain-community==0.0.25?

Comment (bhalder) [reply requirements.txt:3]: is it because we are using latest ragstack-ai?

File: streamlit_app_datastax.py [hunk 2]
```diff
@@ -24,6 +24,18 @@
 from langchain.chains.llm import LLMChain
 from langchain.callbacks.base import BaseCallbackHandler
 
+from langchain_community.callbacks.fiddler_callback import FiddlerCallbackHandler
```
Comment (bhalder) [inline streamlit_app_datastax.py:27]: this will fail without the right lanchain_community version

File: streamlit_app_datastax.py [hunk 3]
```diff
@@ -24,6 +24,18 @@
...
+
+from typing import Any, Dict, List, Optional
+from langchain_core.outputs import LLMResult
+from langchain_community.callbacks.utils import import_pandas
+import time
+
+PROJECT_NAME = "fiddler_chatbot"
+MODEL_NAME = "fiddler_chatbot"
+URL = 'https://preprod.fiddler.ai'
```
Comment (bhalder) [inline streamlit_app_datastax.py:36]: Do we want to use demo instead?

Comment (DannyBrock) [reply streamlit_app_datastax.py:36]: Yes, but demo isn't upgraded yet.

**Pull request merged**